src/python/pants/backend/shell/target_types.py

```diff
@@ -408,7 +408,9 @@ class ShellCommandTarget(Target):
 
 
 class ShellCommandRunTarget(Target):
-    alias = "experimental_run_shell_command"
+    alias = "run_shell_command"
+    deprecated_alias = "run_shell_command"
```

Comment:
Shouldn't this be the old alias?

Reply:
YES. 

grumble

Reply:
Looks like this wasn't fixed before merging? https://github.com/pantsbuild/pants/issues/18301

Reply:
My comment is from after the merge. @chrisjrn fixed in https://github.com/pantsbuild/pants/pull/18267 but that hasn't been merged yet.